stories/Molecules/Text/Authors/Authorcolumn/authors.scss

```diff
@@ -1,63 +1,45 @@
 @import '../../../../assets/scss/variables';
 @import '../../../../assets/scss/breakpoints';
 
-.author-wrapper {
-  border-top: 2px solid $color-black;
-  padding-top: $spacing-32;
-
-  &__box {
-    display: inline-flex;
-    margin-bottom: $spacing-24;
-    width: 100%;
-    
-    &.author-3-col {
-      margin-left: 0.5rem;
-      margin-right: 0.5rem;
-      width: calc(50% - 1rem);
-  
-      .author__label {
-        padding: 0;
-        width: 100%;
-      }
-  
-      .author__img {
-        display: none;
-      }
+.author {
+  &-wrapper {
+    &--1 {
+      width: 100%;
     }
-  
-    @include devicebreak(medium) {
-      margin-left: 0.5rem;
-      margin-right: 0.5rem;
-  
-      &.author-12-col {
-        width: calc(100% - 1rem);
+    &--2 {
+      .author-wrapper__box {
+        width: 50%;
       }
-  
-      &.author-6-col {
-        width: calc(50% - 1rem);
+    }
+    &--3 {
+      .author-wrapper__box {
+        width: 33%;
       }
-  
-      &.author-4-col {
-        width: calc(33% - 1rem);
+    }
+    &--4 {
+      .author-wrapper__box {
+        width: 25%;
```

Comment:
@pallavikadam-srijan It should have some space between the authors and only first and last (according to the grid that if column can have 3 or 4) should remove the spacing.